src/components/NextLink.tsx

```diff
@@ -1,12 +1,11 @@
-/* eslint-disable @typescript-eslint/no-unused-vars */
 import NextLinkOriginal from 'next/link';
-import { Link } from '@taskany/bricks';
+import { Link } from '@taskany/bricks/harmony';
+import { forwardRef } from 'react';
 
-export const NextLink = ({
-    inline,
-    forwardRef,
-    as,
-    ...props
-}: React.ComponentProps<typeof Link> & { forwardRef?: React.Ref<HTMLAnchorElement>; href: string }) => (
-    <NextLinkOriginal {...props} />
+export const NextLink = forwardRef<HTMLAnchorElement, React.ComponentPropsWithoutRef<typeof Link> & { href: string }>(
+    ({ href, ...props }, ref) => (
+        <NextLinkOriginal legacyBehavior passHref href={href}>
```

Comment:
Why u are legacy behavior instead of examples I send?

Reply:
Cuz harmony Link component haven't `as` prop, like legacy component based on `styled-components`